src/app/components/FilterPopup/FilterPopup.jsx

```diff
@@ -359,12 +360,13 @@ class FilterPopup extends React.Component {
         </ul>
       </div>
     );
-    const dateInputError = _map(this.state.raisedErrors, (item) => {
-      if (item.name && item.name === 'date') {
-        return item;
+    const isDateInputError = !!(_find(this.state.raisedErrors, (item) => {
```

Comment:
CLOSE! There's a different method that returns t/f. `find` will return the value itself. And then you're forcing the boolean value (which is okay). Try to find it...